.github/workflows/test.yml

```
@@ -0,0 +1,20 @@
+---
+name: tests
+on: push
+jobs:
+  test:
+    name: Test (Ruby ${{ matrix.ruby }}, ${{ matrix.os }})
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-20.04 ]
+        ruby: [ '2.3', '2.4', '2.5', '2.6', '2.7' ]
```

Is it worth testing against Ruby 3?

```suggestion
        ruby: [ '2.3', '2.4', '2.5', '2.6', '2.7', '3.0' ]
```

---

Yes, but I think we should do that in a dedicated PR. Perhaps work is required.